ddpui/ddpairbyte/schema.py

```
@@ -1,8 +1,243 @@
 from typing import Optional
 from ninja import Schema
 from ddpui.ddpprefect.schema import DeploymentCurrentQueueTime
+from typing import Dict, Any
+from pydantic import Field
+from typing import List, Optional, Dict, Any
```

Ruff (0.12.2)

Redefinition of unused `Optional` from line 1

Remove definition: `Optional`

(F811)